### PR TITLE
Alt+Enter progress to the right cell

### PR DESCRIPTION
**Pull request by se3142**
This pull request implements Alt+Enter to continue editing to the right.
Related issue and change proposal: https://github.com/aardappel/treesheets/issues/139#issuecomment-544167524

Thanks for checking this pull request.

### CODE REVIEW

File: src/document.h [hunk 1]
```diff
@@ -1313,6 +1320,17 @@ struct Document {
                 }
                 return nullptr;
 
+            case A_PROGRESSCELL: {
```
Comment (aardappel) [inline src/document.h:1323]: Looks like this case could be merged with the case above it, with only `A_DOWN` and `A_RIGHT` being different?

File: src/mycanvas.h [hunk 2]
```diff
@@ -114,7 +115,8 @@ struct TSCanvas : public wxScrolledWindow {
         // (scrolling) don't work.
         // The 128 makes sure unicode entry on e.g. Polish keyboards still works.
         // (on Linux in particular).
-        if ((ce.GetModifiers() == wxMOD_ALT) && (ce.GetUnicodeKey() < 128)) {
+        // Special case: ALT+RETURN shall be processed.
+        if (ce.GetKeyCode() != WXK_RETURN && (ce.GetModifiers() == wxMOD_ALT) && (ce.GetUnicodeKey() < 128)) {
```
Comment (aardappel) [inline src/mycanvas.h:119]: `enter` or `alt + enter` is used in the menus?
For consistency, remove () from the latter 2 conditions, or add it to the first

**Review by aardappel — COMMENTED** [review]
Nice! thanks for implementing this!

File: src/document.h (hunk 1 above)
Comment (se3142) [reply src/document.h:1307]: Agreed - both handlers are merged now.

File: src/mycanvas.h (hunk 2 above)
Comment (se3142) [reply src/mycanvas.h]: I revisited the wxWidgets event handling and got to know, that shortcuts are auto-assigned by MyAppend. I simplified my implementation.

**Comment by aardappel** [comment]
Thanks!

**Pull request merged**